spk/syno-magnet/Makefile

```diff
@@ -44,10 +47,24 @@ include ../../mk/spksrc.extract.mk
 
 .PHONY: syno_magnet_copy
 
-# Icons already included in download
-$(ICON_DIR): ;
+# Override because we need type: legacy
```

Comment:
Please add SERVICE_TYPE support with default value "url" in spk.service.mk directly at https://github.com/SynoCommunity/spksrc/blob/master/mk/spksrc.service.mk#L214

Reply:
I changed my mind about variable name, SERVICE_URL_TYPE seems more appropriate to me.

Reply:
Correct.